test/Tests/src/Data/Text_Spec.enso

```diff
@@ -1124,9 +1141,9 @@ spec =
 
         Test.specify "locate should work as shown in examples" <|
             example_1 =
-                "Hello World!".locate "J" == Nothing
-                "Hello World!".locate "o" == Span.Value (4.up_to 5) "Hello World!"
-                "Hello World!".locate "o" mode=Matching_Mode.Last == Span.Value (4.up_to 5) "Hello World!"
+                "Hello World!".locate "J" . should_equal Nothing
+                "Hello World!".locate "o" . should_equal Span.Value (4.up_to 5) "Hello World!"
+                "Hello World!".locate "o" mode=Matching_Mode.Last . should_equal Span.Value (4.up_to 5) "Hello World!"
```

Comment:
I think we need a set of tests for `index_of` once it has been added to the Text API. We can probably re-use some locate tests.

Reply:
Yes forgot to add these.
Also Example 1 tests were a load of rubbish before